pkg/vm/engine/disttae/logtailreplay/rows_iter.go

```diff
@@ -51,6 +51,23 @@ func (p *PartitionState) NewRowsIter(ts types.TS, blockID *types.Blockid, iterDe
 	return ret
 }
 
+func (p *PartitionState) NewDirtyRowsIter(
+	ts types.TS,
+	blockID *types.Blockid,
+	iterDeleted bool) *rowsIter {
```

Comment:
rows in dirtyRows are always deleted so this param may be removed.

Reply:
yes